Gem/Code/Source/Weapons/SceneQuery.cpp

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#include <Source/Weapons/SceneQuery.h>
+#include <AzFramework/Physics/Common/PhysicsSimulatedBody.h>
+#include <AzFramework/Physics/ShapeConfiguration.h>
+#include <AzFramework/Physics/PhysicsScene.h>
+#include <AzFramework/Physics/Material.h>
+#include <Multiplayer/NetworkEntity/INetworkEntityManager.h>
+
+namespace MultiplayerSample
+{
+    namespace SceneQuery
+    {
+        static AZStd::shared_ptr<Physics::ShapeConfiguration> GatherShapeToPhysicsShape(const GatherShape& gatherShape)
+        {
+            const float defaultValue = 1.0f; // TODO: Pass via gather params
+
+            switch (gatherShape)
+            {
+            case GatherShape::Box:
+                return AZStd::make_unique<Physics::BoxShapeConfiguration>(AZ::Vector3(defaultValue));
+                break;
+            case GatherShape::Sphere:
+                return AZStd::make_unique<Physics::SphereShapeConfiguration>(defaultValue);
+                break;
+            case GatherShape::Capsule:
+                return AZStd::make_unique<Physics::CapsuleShapeConfiguration>(defaultValue, defaultValue);
+                break;
+            default:
+                AZ_Warning("", false, "Only box, sphere, and capsule conversions are supported.");
+                return nullptr;
+            }
+
+        }
+
+        static AzPhysics::SceneQuery::QueryType GetQueryTypeFromFilter(const IntersectFilter& filter)
+        {
+            // There's no "None" type for the scene query type, using "Static" by default.
+            AzPhysics::SceneQuery::QueryType queryType = AzPhysics::SceneQuery::QueryType::Static;
```

Comment:
Maybe just delete the HitStatic and HitDynamic enums and use AzPhysics::SceneQuery::Query type directly instead? Doesn't make sense to have enums that wrap an enum with conversion functions..